styles/style.scss

```diff
@@ -0,0 +1,166 @@
+
+*{
+    box-sizing: border-box ;
+}
+.container{
+  position: relative;
+ 
+}
+.container_img{
+    width: 100%;
+    height: 550px;
+}
+.container_playbtn{
```

Comment:
@sandesh9988 

Don't name classes like this. It should be playbutton. Not, "container_playbtn"

Single underscore means nothing. If you're writing BEM, it should be double underscores.